OpenRA.Mods.Common/Scripting/Properties/AircraftProperties.cs

```diff
@@ -16,12 +16,13 @@
 
 namespace OpenRA.Mods.Common.Scripting
 {
+	[ExposedForAIScript]
```

Comment:
I tried to that but I got lost. The only place I saw per method level operation was in Lua doc generation and ActorCommands and PlayerCommands seemed to work on class level. Can you detail it please?